SmartEVSE2.X/EVSE.c

```diff
@@ -1425,8 +1425,8 @@ unsigned int getItemValue(unsigned char nav) {
  * @param unsigned char nav
  * @return unsigned char[] MenuItemOption
  */
-const far char * getMenuItemOption(unsigned char nav) {
-    unsigned char Str[10];
+const char * getMenuItemOption(unsigned char nav) {
+    static unsigned char Str[10];
```

Comment:
added static qualifier as this pointer is used as a return value and could be destroyed after return

Reply:
Exactly: https://github.com/SmartEVSE/SmartEVSE-2/pull/6/commits/5814af6280e51cbbfc12d920fbc62411f54a0045#diff-3f22bc2aa11b85071fa00c848a58a870628ef4428ebec048596213bd13831780R1439
:)

Reply:
And you have the better comment!

Reply:
Thanks , both!